src/daemon/device_mouse.c

```diff
@@ -75,6 +75,25 @@ int setactive_mouse(usbdevice* kb, int active){
     return 0;
 }
 
+// FIXME: every time the battery is read, the mouse doesn't accept input, which makes using it quite uncomfortable to use
```

Comment:
Has this been remedied or worked around? I can see us getting a *bunch* of issues about the mouse not working due to this.

Reply:
Yes. This is the issue with DELAY_MEDIUM.

Reply:
If so, could you remove the FIXME comment?

Reply:
It's still there because the fix isn't committed yet. I'll add it and remove the comment.